packages/database/src/core/view/QueryParams.ts

```diff
@@ -178,7 +174,18 @@ export class QueryParams {
     return this.index_;
   }
 
-  private copy_(): QueryParams {
+  loadsAllData(): boolean {
+    return !(this.startSet_ || this.endSet_ || this.limitSet_);
+  }
+
+  /**
+   * What does isDefault() mean?
```

Comment:
Were you planning to get an answer before checking in the code?

Reply:
Removed the comment. But what does it mean @schmidt-sebastian ?

Reply:
It's a query with no filters and no constraints with the default ordering (by priority)